backend/src/Squidex.Domain.Apps.Entities.MongoDb/Contents/Operations/QueryByQuery.cs

```diff
@@ -10,7 +10,9 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using MongoDB.Bson.Serialization;
```

Comment:
Please check the usings and remove unnecessary usings. Ensure that there are no compiler warnings.